fixtures/webstudio-remix-netlify-edge-functions/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "scripts": {
+    "predev": "node -e \"fs.rmSync('./public/_redirects', { recursive: true, force: true })\"",
+    "start": "netlify serve",
+    "build": "remix build",
+    "dev": "remix dev",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@netlify/functions": "^1.3.0",
+    "@netlify/edge-functions": "^2.0.0",
+    "@netlify/remix-edge-adapter": "1.2.0",
+    "@remix-run/react": "^1.19.2",
+    "@remix-run/server-runtime": "^1.19.2",
+    "@remix-run/node": "^1.19.2",
+    "@webstudio-is/react-sdk": "workspace:*",
+    "@webstudio-is/sdk-components-react-radix": "workspace:*",
+    "@webstudio-is/sdk-components-react-remix": "workspace:*",
+    "@webstudio-is/sdk-components-react": "workspace:*",
+    "@webstudio-is/form-handlers": "workspace:*",
+    "@webstudio-is/image": "workspace:*",
+    "@webstudio-is/sdk": "workspace:*",
+    "isbot": "^3.6.8",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "@netlify/remix-adapter": "^1.0.0",
+    "cross-env": "^7.0.3"
+  },
+  "private": true,
+  "sideEffects": false,
+  "devDependencies": {
+    "@remix-run/serve": "^1.19.2",
+    "@remix-run/dev": "^1.19.2",
+    "@types/react": "^18.2.20",
+    "@types/react-dom": "^18.2.7",
+    "@webstudio-is/http-client": "workspace:*",
+    "eslint": "^8.48.0",
```

Comment:
Eslint again

Reply:
We actually discussed about that, but didn't remove from the `templates`. Removing it now 👍